agent/firewall.go

```diff
@@ -113,23 +114,24 @@ func (fw *Firewall) Init(netif NetIf) error {
 	// Allow ICMP, DHCP and SSH between host and instances.
 	hostAddr := fw.Agent.networkConfig.romanaGW
 	inputRules := []string{
-		fmt.Sprintf("-d %s/32 -p icmp -m icmp --icmp-type 0 -m state --state RELATED,ESTABLISHED", hostAddr),
-		fmt.Sprintf("-d %s/32 -p tcp -m tcp --sport 22", hostAddr),
 		"-d 255.255.255.255/32 -p udp -m udp --sport 68 --dport 67",
```

Comment:
This is for DHCP, right? But it isn't needed for Kubernetes, yet we are creating those rules even there.

Ideally, we remove even this and then have deployment specific default policies specified, which are applied through the policy manager.